Objects/longobject.c

```diff
@@ -115,6 +115,38 @@ maybe_small_long(PyLongObject *v)
         if (PyErr_CheckSignals()) PyTryBlock    \
     } while(0)
 
+#define PYLONG_FROM_SIGNED(INT_TYPE, ival) \
```

Comment:
What has been changed from the original code? If nothing has changed except that it was written in a macro form, then I don't think we should have this refactorization in this PR.